libs/wingsdk/src/target-tf-aws/react-app.ts

```diff
@@ -15,7 +16,7 @@ import * as ex from "../ex";
  */
 export class ReactApp extends ex.ReactApp {
   private _host: cloud.Website;
-  constructor(scope: Construct, id: string, props: ex.ReactAppProps) {
+  constructor(scope: Construct, id: string, props: aws.AwsReactAppProps) {
```

Comment:
following #4543, can you please change it to `AwsReactAppOptions`?  (also the other props if possible 🙏 )

Reply:
@tsuf239 

I follow this [PR](https://github.com/winglang/wing/pull/4740) and this [issue](https://github.com/winglang/wing/issues/4543)